client/searchResults.js

```diff
@@ -0,0 +1,121 @@
+const searchBar = document.querySelector('#searchbartext');
+
+searchBar.addEventListener('keypress', e => {
+    if(e.key === 'Enter' && e.target.value) {
+        window.location.search = `?q=${encodeURIComponent(e.target.value)}`;
+    }
+});
```

Comment:
Took a while but think this is a workaround for the form submit behaviour not liking buttons (at least that is what happened for us) if so, nice solution